Berlin/BerOr1270.xml

```diff
@@ -24,16 +30,265 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+<decoNote type="ornamentation" xml:id="d6">
+<locus target="#25r"></locus>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d7">
+<locus target="#28r"></locus>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d8">
+<locus target="#30ra"></locus>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d9">
+<locus target="#33vb"></locus>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d10">
+<locus target="#35ra"></locus>
+</decoNote>
+</decoDesc>
+<additions>
+<list>
+  <item xml:id="a1">
+<desc type="Record">Calculation of the jubilee years</desc>
+<note>Recent hand; crudely written; black and red ink.</note>
+<locus from="24va" to="24vb"></locus>
+<q xml:lang="gez">ግጽው፡ ዘኢዮቤልዬ ... </q>
+  </item>
+  <item xml:id="a2">
+<desc type="Inventory">Inventory of the lands</desc>
+<note>Crudely and recently written; black ink.</note>
+<locus from="26ra" to="27rb"></locus>
+<q xml:lang="gez">ኆልቈ፡ ገራሕቱ፡ ... </q>
+  </item>
+  <item xml:id="a3">
+<desc type="Excerpt">Prayer of the incense</desc>
+<note>Written in black ink. The text on f. 39r is within any columns (f. 39r is without layout). The handwriting seems to be slightly different, but not so much recent than the main hand (in a smaller size).</note>
+<locus from="38vb" to="39r"></locus>
+  </item>
+  <item xml:id="a4">
+<desc type="Unclear">Unclear note</desc>
+<note>Crudely and recently written (after Additio 3); black ink. The initial lines have been erased.</note>
+<locus target="#39r"></locus>
+  </item>
+  <item xml:id="a5">
+    <note>Black ink. Recent hand.</note>
+    <locus target="#39v"></locus>
+<desc type="Supplication"></desc>
+<q xml:lang="gez">መስተብቍዕ፡...</q>
+</item>
+<item xml:id="e1">
+  <desc type="Correction">Scribal Corrections: few omitted letters written interlineally (e.g.<locus target="#11va"></locus>, <locus target="16vb"></locus>).</desc>
+</item>
+<item xml:id="e2">
+<desc>Writing exercises on <locus target="#1r"></locus>, <locus target="#10v"></locus> (in the margins).</desc>
+</item>
+<item xml:id="e3">
+<desc type="StampExlibris"> on <locus target="#1r"></locus> and on <locus target="#40v"> (at the bottom).</locus></desc>
+</item>
+<item xml:id="e4">
+<desc type="Unclear">Text not readable on <locus from="40v" to="41r"></locus>
+</desc>
+</item>
+</list>
+</additions>
+<bindingDesc>
+<binding contemporary="true" xml:id="binding">
+<decoNote xml:id="b1">European cardboard binding, with two white paper leaves are at the front and back side, as protective leaves of the text block; recent textile inlays and sewing used as reinforcements of the quires; original boards and covers missing.</decoNote>
+<decoNote xml:id="b2" type="bindingMaterial">
+<material key="cardboard"></material>
+<material key="paper"></material>
+<material key="textile"></material>
+</decoNote>
+</binding>
+</bindingDesc>
+</physDesc>
+<history>
+<origin>
+<origDate notBefore="1350" notAfter="1450">Mid-14th mid-15th century</origDate>
+<note>Dating based on paleographical grounds.</note>
```

Comment:
I think that you can simply delete this `<note>` and instead add the attribute evidence="lettering" to the element origDate

Reply:
I did, thank you.